index.html

```diff
@@ -5,6 +5,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.4/css/bulma.min.css">
         <link rel="stylesheet" type="text/css" href="./assets/css/main.css">
+         <link rel="stylesheet" type="text/css" href="./assets/css/scrollbar.css">
```

Comment:
Scrollbar properties are added but no selector is matched in the html. 
I suggest to add a class to the root element, so the scrollbar will be included like this:
`<div id="app" class="scrollbar">`